CHANGELOG.md

```diff
@@ -35,6 +35,63 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Security
 
+## [0.9.4] - 2023-04-20
+
+### Added
+- Adds ability to pipe queries to the CLI
+- Adds ability to run PartiQL files as executables by adding support for shebangs
+
+### Changed
+
+### Deprecated
+
+### Fixed
+- Fixes list/bag ExprValue creation in plan evaluator
+- Fixes gradle build issues.
```

Comment:
Do we want to remove those? Those reads as in addition to "Revert changes in 0.9.3 (the previous version) as they include breaking changes", we also have those included in the release.

Reply:
I think these were part of v0.9.2. See https://github.com/partiql/partiql-lang-kotlin/blob/eb013103e0f11a100ab3d00f28a9584388c5502f/CHANGELOG.md#092---2023-01-20